src/test/TestUtils/TestProjectFixture.cs

```diff
@@ -310,15 +317,10 @@ public TestProjectFixture EnsureRestored(params string[] fallbackSources)
 
         public TestProjectFixture EnsureRestoredForRid(string rid, params string[] fallbackSources)
         {
-            var sourceProjectJson = Path.Combine(_testProject.ProjectDirectory, "project.json.template");
-            var targetProjectJson = Path.Combine(_testProject.ProjectDirectory, "project.json");
-
-            // apply RID to template
-            File.WriteAllText(targetProjectJson, File.ReadAllText(sourceProjectJson).Replace("{RID}", rid));
-
             if ( ! _testProject.IsRestored())
             {
-                RestoreProject(fallbackSources);
+                string extraMSBuildProperties = $"/p:RuntimeIdentifiers={rid} /p:RuntimeIdentifier={rid}";
```

Comment:
Actually -- I had already made that change part of the way in the project file. I'm just passing one property through here now.